inst/rmarkdown/templates/ams_article/resources/template.tex

```diff
@@ -0,0 +1,70 @@
+\documentclass[$col$]{ametsoc}
```

Comment:
I don't see any of the pandoc includes in this template (e.g. `include-header`, `include-before`, `include-after`). Should these be added?